Sukyung/0412/BJ18808.java

```diff
@@ -0,0 +1,108 @@
+package day0412;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class BJ18808 {
+	static int N, M, K, R, C;
+	static int[][] notebook;
+	static int[][] sticker;
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer st = new StringTokenizer(br.readLine(), " ");
+		N = Integer.parseInt(st.nextToken());
+		M = Integer.parseInt(st.nextToken());
+		K = Integer.parseInt(st.nextToken());
+		notebook = new int[N][M];
+
+		for (int k = 0; k < K; k++) {
+			st = new StringTokenizer(br.readLine(), " ");
+			R = Integer.parseInt(st.nextToken());
+			C = Integer.parseInt(st.nextToken());
+			sticker = new int[R][C];
+
+			for (int i = 0; i < R; i++) {
+				st = new StringTokenizer(br.readLine(), " ");
+				for (int j = 0; j < C; j++) {
+					sticker[i][j] = Integer.parseInt(st.nextToken());
+				}
+			}
+			if (!attach()) { // 0
+				for (int i = 0; i < 3; i++) { // 90, 180, 270
+					swap();
+					rotate();
+					if (attach())
+						break;
+				}
+			}
```

Comment:
```suggestion
				for (int i = 0; i < 4; i++) { // 90, 180, 270
					if (attach())
						break;
					swap();
					rotate();
				}
```
이렇게 짜는편이 좀 더 좋았을 것 같습니다

Reply:
피드백 감사합니다🙂